README.md

```diff
@@ -1,12 +1,24 @@
+![MageLogo](https://user-images.githubusercontent.com/99209078/171298628-d8f34d5b-9771-404d-947c-f446ce083215.png)
 # Mage AI
+Mage AI cleans your data using a single Python library.
 
 ## Table of Contents
 1. [Introduction](#introduction)
 1. [Installation](#installation)
 1. [Getting Started](#getting-started)
+2. [Usage](#usage)
+3. [To Contribute](#to-contribute)
 
 ## Introduction
-Mage AI is a Python library for data cleaning.
+Mage AI aims to simplify data pre-processing. With a few lines of code, you can:
+- Generate data cleaning suggestions in your local notebook
+- Display feature-specific statistics and visualizations
+- Easily detect "holes" and anomalies in your data
```

Comment:
Create data cleaning pipeline and clean your data with a few clicks.

Reply:
Xiaoyou, are you suggesting I replace lines 13 -16 with a single sentence "Create data cleaning pipeline and clean your data with a few clicks." or adding that sentence as a list item?